### DISCUSSION TITLE
Deep Sleep

### DISCUSSION THREAD

**Discussion by gekkoton** [darthcloud/BlueRetro#453] [Q&A] [answered]
Hi guys! In the system macro section it mentions the Deep sleep button combination as:

Deep Sleep: Main Left Trigger + Main Right Trigger + Middle Right (Start) + Face Up + D-pad Down

What the 'Face up' button corresponds to?

**Comment by darthcloud** [comment, marked as answer]
Macro doc now include PlayStation buttons reference
https://github.com/darthcloud/BlueRetro/wiki#5---button-combinations-functions

You can also use the mapping reference
https://docs.google.com/spreadsheets/d/e/2PACX-1vT9rPK2__komCjELFpf0UYz0cMWwvhAXgAU7C9nnwtgEaivjsh0q0xeCEiZAMA-paMrneePV7IqdX48/pubhtml

**Reply by gekkoton** [reply]
That was very helpful, thanks a lot darthcloud!
